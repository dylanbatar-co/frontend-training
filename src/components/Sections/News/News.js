import React from "react";

export const News = () => {
  return (
    <section>
      <h2>Top news</h2>
    </section>
  );
};
