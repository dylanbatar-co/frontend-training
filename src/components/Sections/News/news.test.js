import { shallow } from 'enzyme';
import { News } from './News';

describe('News section Test', () => {
  it('News component snapshot', () => {
    const component = shallow(<News />);
    expect(component).toMatchSnapshot();
  });
});
