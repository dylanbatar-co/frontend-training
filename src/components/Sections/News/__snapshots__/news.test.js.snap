// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News section Test News component snapshot 1`] = `
<section
  className="news-grid container"
>
  <h2
    className="section-title mb-4"
  >
    Top news
  </h2>
  <NewsGrid />
</section>
`;
